packages/react/src/components/List/List.tsx

```diff
@@ -218,6 +218,12 @@ class List extends AutoControlledComponent<WithAsProp<ListProps>, ListState> {
 }
 
 /**
- * A list displays a group of related content.
+ * A List displays a group of related sequential items.
```

Comment:
This section a bit confusing for me in a sense what we offer. Currently, we have just static list (1st option) and navigable selectable list (2nd option). 
Currently, navigability can be achieved by specifying `selectable` prop and it will set `listbox` role out of the box, which indicated a selectable list semantic.
To achieve 3rd option, external tweaks are required, for e.g. specifying `selectable` prop but overriding `listbox` to `list` to get rid of 'selectable' semantic, what we don't offer

Reply:
so, would you suggest to remove `Selectable List` from the set of options, so that we will only target scenarios that are easy (and, what is more important, _safe_) to achieve out-of-the-box?

Reply:
I meant to leave what we support now
- static non-navigable list
- selectable navigable list

remove 3rd option - "navigable actionable list", for now. But this is something to discuss - having a navigable list (not as selectable) with actionable items